stack/building-pipelines-lsst-io-locally.rst

```diff
@@ -20,6 +20,10 @@ Before starting, you’ll need a working ``lsst_distrib`` installation.
 
 This installation needs to be a recent daily or weekly build so that any in-development packages will compile with the Stack. Working from the tip of the ``master`` branch is the norm for LSST software development.
 
+Finally, the documentation build uses Documenteer_ and related Sphinx_ documentation packages.
+Documenteer_ is already installed if you are using the Rubin Conda environment (part of the usual Science Pipelines installation).
```

Comment:
This text is a bit problematic, as the documenteer version installed with the stack is not yet operational for building pipelines.lsst.io, as I understand it.  We are still building in Jenkins with 0.5.x from `requirements.txt`, not 0.6.x from conda, although we're close to changing over.